packages/info/README.md

```diff
@@ -9,22 +9,73 @@ This package returns a set of information related to the local environment.
 ## Installation
 
 ```bash
+#npm
 npm i -D @webpack-cli/info
+
+#yarn
+yarn add @webpack-cli/info -D
 ```
 
 ## Usage
 
+### Args / Flags
+
+#### Output format
+
+| Flag                | Description                   | Type        |     |     |
+| ------------------- | ----------------------------- | ----------- | --- | --- |
+| `--output-json`     | To get the output as JSON     | [ boolean ] |     |     |
+| `--output-markdown` | To get the output as markdown | [ boolean ] |     |     |
+
+#### Options
+
+| Flag                | Description                                                     | Type        |
+| ------------------- | --------------------------------------------------------------- | ----------- |
+| `--help`            | Show help                                                       | [ boolean ] |
+| `--version`         | Show version number of `webpack-cli`                            | [ boolean ] |
+| `--system` , `-s`   | System information ( OS, CPU )                                  | [ boolean ] |
+| `--binaries` , `-b` | Installed binaries (Node, yarn, npm)                            | [ boolean ] |
+| `--browsers`        | Installed web browsers                                          | [ boolean ] |
+| `--npmg`            | Globally installed NPM packages ( webpack & webpack-cli only )  | [ boolean ] |
+| `--npmPackages`     | Info about packages related to webpack installed in the project | [ boolean ] |
+
 ### Node
 
 ```js
-const envinfo = require("@webpack-cli/info").default;
-envinfo();
+const info = require("@webpack-cli/info").default;
+
+async function wrapperFunc() {
+	await info({
+		/* Custom Config */
+	});
+}
+wrapperFunc();
 ```
 
+#### Custom config
+
+> Config has higher precedence than system flags
+
+```json
+{
+    // Config relative path
```

Comment:
JSON format can't have comments. If you want have comments, could you change format?

Reply:
I know JSON doesn't have comments, it for demonstration purpose only. Will change the format.